apps/etterlatte-saksbehandling-ui/client/src/components/behandling/trygdetid/TrygdetidGrunnlag.tsx

```diff
@@ -116,7 +116,7 @@ export const TrygdetidGrunnlag: React.FC<Props> = ({
 
             <FormWrapper>
               <Begrunnelse
-                value={trygdetidgrunnlag.begrunnelse}
+                value={trygdetidgrunnlag.begrunnelse ?? ''}
```

Comment:
Blir dette riktig? denne blir vel kanskje lagret som tom string i stedet for `null` nå?

Reply:
blir riktig det